{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": null,
    "doc_source_url": null,
    "doc_url": null,
    "home": "https://github.com/pytest-dev/pytest-faulthandler",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "MIT",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/p/pytest-faulthandler/pytest-faulthandler-1.5.0.tar.gz",
    "subdirs": [
      "linux-64"
    ],
    "summary": "py.test plugin that activates the fault handler module for tests",
    "tags": null,
    "text_prefix": false,
    "timestamp": 1538751194,
    "version": "1.5.0"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main"
  ],
  "package": "pytest-faulthandler-1.5.0-py37_1000.tar.bz2",
  "repodata": {
    "build": "py37_1000",
    "build_number": 1000,
    "depends": [
      "pytest",
      "python >=3.7,<3.8.0a0"
    ],
    "license": "MIT",
    "md5": "f0b2cbf6b34c83fc993a66317aabd400",
    "name": "pytest-faulthandler",
    "sha256": "2de03b56b561b46c26fcf0855c9bc08af3e0dc97ca626484636c33cd839f5cc0",
    "size": 7870,
    "subdir": "linux-64",
    "timestamp": 1538751194582,
    "version": "1.5.0"
  },
  "repodata_version": 1,
  "subdir": "linux-64",
  "url": "https://conda.anaconda.org/conda-forge/linux-64/pytest-faulthandler-1.5.0-py37_1000.tar.bz2"
}